Update EnableBranchingModal.tsx (#19736)
supabase/supabase: apps/docs/pages/guides/platform/branching.mdx

You can also combine both strategies to perform both local and staging tests.

## Early Access Pricing

> Branching is available for Early Access, available on the Pro Plan and above.
> 
> The Early Access pricing is as follows:
> 
> - Each Preview branch is $0.32 per day
> - Each Preview branch is billed until it is removed

Reviewer: ```suggestion
Branching is available on the Pro Plan and above for Early Access users. The price during Early Access is:

- Each Preview branch costs $0.32 per day
- Each Preview branch is billed until it is removed
```
